static/app/views/traces/styles.tsx

```diff
@@ -67,22 +68,23 @@ export const SpanTablePanelItem = styled(StyledPanelItem)`
 export const BreakdownPanelItem = styled(StyledPanelItem)<{highlightedSliceName: string}>`
   ${p =>
     p.highlightedSliceName
-      ? `--highlightedSlice-${p.highlightedSliceName}-opacity: 1.0;
-         --highlightedSlice-${p.highlightedSliceName}-saturate: saturate(1.0) contrast(1.0);
-         --highlightedSlice-${p.highlightedSliceName}-transform: translateY(0px);
+      ? css`
+          --highlightedSlice-${p.highlightedSliceName}-opacity: 1;
+          --highlightedSlice-${p.highlightedSliceName}-saturate: saturate(1) contrast(1);
+          --highlightedSlice-${p.highlightedSliceName}-transform: translateY(0px);
        `
       : null}
   ${p =>
     p.highlightedSliceName
-      ? `
-        --defaultSlice-opacity: 1.0;
```

Comment:
hmmm it did this automatically

Reply:
seems fine

Reply:
It didn't know what to do before since it wasn't in the tagged template